assets/javascripts/modules/autoComplete.js

```diff
@@ -369,6 +380,8 @@ var getSuggestions = function (el) {
     // generate a global variable to hold the data and insert into data-suggestions attribute
     window.hmrcAutocompleteData = suggestions;
     el.attr("data-suggestions", "hmrcAutocompleteData");
+    // update text input with a pre-selected select option
+    loadExistingValue();
```

Comment:
A thought would this be better sat in `setup`? 

Potentially the variable `dataLinked` needs to become a file variable?